### DISCUSSION TITLE
Is there a way to know wether a blocker is active?

### DISCUSSION THREAD

**Discussion by GabriFila** [TanStack/router#2158] [Q&A]
Hello, amazing library!

Suppose I'm blocking the navigation using the `useBlocker` hook, is there a way to know, in a different part of the UI, that a blocker is active anywhere in the app? I would like to auto reload my application without doing it while the user is performing a blocking action.

Thank you!

**Comment by schiller-manuel** [comment]
wouldn't the blocker prevent the "reload" already? or what does "reload" really mean here?
